test/resources/knativeserving.go

```diff
@@ -36,20 +38,78 @@ import (
 
 const (
 	interval = 10 * time.Second
-	timeout  = 2 * time.Minute
+	timeout  = 5 * time.Minute
 )
 
+// CreateKnativeServingReady polls the status of the KnativeServing called name
+// from client every `interval` until `inState` returns `true` indicating it
+// is done, returns an error or timeout.
+func CreateKnativeServingReady(t *testing.T, clients *test.KnativeServingAlphaClients, names test.ResourceNames,
+	inState func(s *v1alpha1.KnativeServing, err error) (bool, error)) (*v1alpha1.KnativeServing, error) {
+	svc, err := CreateKnativeServing(t, clients, names)
```

Comment:
I would move this out of here. Since all the tests require a KS, just have this test verify a Ready state, assuming the KS has already been created (see above).